### ISSUE TITLE
RuntimeError: bool value of Tensor with more than one value is ambiguous

### ISSUE THREAD

**Issue by LemonMi** [nearai/torchfold#8]
if args not in self.cached_nodes[op]:
RuntimeError: bool value of Tensor with more than one value is ambiguous

This error appeared at every time. This time i run my code until the 16th epoch. Do you have any idea? Thank you very much.

**Comment by supriya-gdptl** [comment]
I am also facing the same issue.
Could you please help? @ilblackdragon @nearmax

Thank you

**Comment by ilblackdragon** [comment]
@supriya-pandhre Sorry, we are not actively maintaining this repo right now as we shifted our work to https://github.com/nearprotocol development.

Probably the error is around the fact that you can not do `in` with tensors anymore. The solution is to change to `not any([arg == x for x in self.cached_nodes[op]])` or something of this sort. I sadly don't even have pytorch installed right now so won't  be able to repro. But if you make a PR with the test that fails and a fix I'll review it.

**Comment by LemonMi** [comment]
@supriya-pandhre  Have you solved this problem? I'm still troubled with this issue..

**Comment by supriya-gdptl** [comment]
@LemonMi No, I tried to fix it, but still getting the same error. Looks like I need to write code without torchfold (shouldn't be that difficult, hopefully)
